client/src/lib/Form.svelte

```diff
@@ -21,47 +24,54 @@
   }
 
   async function request(address: string): Promise<string> {
-    return faucetRequest(address, token);
+    return faucetRequest(address, token, network);
   }
 </script>
 
-
 <form on:submit|preventDefault={onSubmit} class="w-full">
-  <label class="label" for="address">
-    <span class="label-text">Your SS58 Address</span>
-  </label>
-  <input
-    type="text"
-    bind:value={address}
-    placeholder="Enter your address"
-    class="input w-full input-primary mb-6 text-sm"
-    id="address"
-    disabled={!!webRequest}
-  />
+  <div class="inputs-container">
+    <label class="label" for="address">
+      <span class="label-text">Your SS58 Address</span>
+    </label>
+    <input
+      type="text"
+      bind:value={address}
+      placeholder="Enter your address"
+      class="input w-full input-primary text-sm"
+      id="address"
+      disabled={!!webRequest}
+    />
+  </div>
+  <div class="inputs-container md:grid md:grid-cols-3 md:gap-4 ">
+    <div class="form-control">
+      <label class="label cursor-pointer">
+        <span class="label-text">Use parachain</span>
+        <input type="checkbox" bind:checked={useParachain} class="checkbox checkbox-primary"/>
+      </label>
+    </div>
+    <div class="form-control w-full max-w-xs col-span-2">
+      <input disabled={!useParachain} bind:value={network}
```

Comment:
i guess next PRs when we implement multinetwork, this will be renamed to `bind:value={parachain_id}`, and network will be a list with { rococo, westend }